app/assets/javascripts/initializers/initializeUserProfilePage.js

```diff
@@ -33,3 +33,18 @@ function initializeUserProfilePage() {
     }
   }
 }
+
+function initializeProfileInfoToggle() {
```

Comment:
I don't understand how to trigger this :|

Reply:
It's being added here: https://github.com/forem/forem/pull/11357/files#diff-16747096d6c3ff6309080c7463dca64c498ec9c117509745d9999e0f85119b86R60

I don't think this is the ideal way of running/triggering JS but it seems like it's a commonly used method on DEV so I just sticked to it.

Reply:
No I mean, as a user, how to I trigger that? I tried clicking on it but nothing happens